src/languages/typescript.js

```diff
@@ -222,6 +223,14 @@ export default function(hljs) {
       {
         begin: '\\.' + hljs.IDENT_RE, relevance: 0 // hack: prevents detection of keywords after dots
       },
+      {
+        begin: '\\b' + IDENT_RE + ':', relevance: 0 // prevents detection of keywords in property name
+      },
+      {
```

Comment:
I don't think we can add this.  Many of our built-ins ARE function names (all BUILT_IN_GLOBALS for example)... what specific issues are you having that caused you to add this rule?

Reply:
```
yyyc514 21 hours ago Member
Please show an example or two of exactly what you’re trying to avoid by adding this.

Lhoerion 21 hours ago Author
Pretty straightforward example:
public readonly type: number;

yyyc514 21 hours ago Member
I’m thinking this should probably reuse ECMAScript.IDENT_RE, isn’t that what that is?... you can use regex.concat to glue the pieces together.

Lhoerion 21 hours ago Author
Replaced it with simple concatenation
```

Reply:
I'm referring to the block STARTING on line 229...  not the proceeding.

Reply:
How annoying how GitHub did this. Let me try again.